js_functions/post.js

```diff
@@ -0,0 +1,13 @@
+export default function postData(path, data) {
+  const options = {
+    method: "POST",
+    headers: { "content-type": "application/json"},
+    body: JSON.stringify(data),
+  };
+
+  fetch(`https://boolean-api-server.fly.dev/Chloe0701962/${path}`, options)
+    .then((r) => r.json())
+    .catch((error) =>
+      console.log("An error occured while fetching data:", error)
```

Comment:
Doing `console.error("")` will highlight the error as red in the console to make these easier to spot